formats/capabilities/SPEC_1.0.md

```diff
@@ -0,0 +1,91 @@
+
+`/capabilities/1.0/private/{encrypt(receiver, concat(receiver-pubkey, nonce))}/{id}`
+
+## Capability document IDs
+
+You may want one, or many, depending on the application, if you want to coordinate between multiple client devices etc. These are opaque to the receiver, who should decode all documents under the path prefix.
+
+A suggested scheme for this would be using `/1`, `/2` etc. If you're merely extending the lifetime of an existing cap, then you can write to the same path. It doesn't matter if you have two clients that independently do this. If you want to change the contents, you should increment the number, and tombstone older entries. This behaviour doesn't handle the same user making changes to the same receiver's caps concurrently on two devices they own - only one will win.
+
+## Payload format
+
+These should be encrypted under the pubkey of the receiver of the capability. This prevents anyone from knowing which Namespace/Subspace/Path the capability is for
+
+This should be a list of `McCapability` (TODO whatever the standard encoding of this is)
+
+## Use cases
+
+### week-by-week caps
+
+Imagine you want to grant someone access to your social media posts (e.g. a private Twitter account), but you want to be able to revoke this in future. Granting someone an infinite length of time in a capability prevents you from ever being able to revoke this access. But while giving someone a capability document once is feasible, being able to continually do this while you're both offline is a challenge.
+
+The solution is to grant two capabilities, first:
+
+```
+CommunalCapability:
+  access_mode: read
+  namespace_key: gardening.xxx...
+  user_key: UserPublicKey
+  delegations:
+  - times: forever
+    path: /capabilities/1.0/private/{their-bit}...
+    subspace: you
+```
+
+This will allow them forever access - but only to capabilities specific to them.
+
+Then you can put the actual week by week capability in there. Each time you make an entry, or regularly on expiry (whichever makes sense for your app), overwrite the entry at that path with a new capability, extending the time. This prevents there being lots of documents, and the first thing they can do on sync is get the latest capability, and then use that to sync all entries in the past. This could likely be done by the same client you're using to make the entries - if you aren't making you entries, the existing caps are sufficient. If you make an entry outside of the current end time, you'll want to update them all. Any peer will get the updated caps in the same sync as the new entries, whether that's a direct peer or via some pub/relay.
+
+To stop someone accessing future Entries, stop updating the capability (and optionally tombstone the old one).
+
+### Granting public access
```

Comment:
Can probably omit the details for this from the design process until everything else has settled.